can_bus/quadruna/BMS/BMS_tx.json

```diff
@@ -388,5 +388,18 @@
                 "bits": 16
             }
         }
+    },
+    "BMS_Shdn_Node_Status": {
```

Comment:
Can message/signal names should be `PascalCase`. I'm fine to forego this for the net names since those are the real names, but this message should be `BmsShdnNodeStatus`